README.md

```diff
@@ -186,6 +186,11 @@ Windows VMs. The private key should be in PEM encoded RSA format.
 oc create secret generic cloud-private-key --from-file=private-key.pem=$HOME/.ssh/$keyname
 ```
 
+If you do not have a PEM encoded private key in RSA format, you can generate one using:
+```shell script
+openssl genrsa -des3 -out <KEY_NAME>.pem 2048
```

Comment:
RHEL docs usually recommend `ssh-keygen -t rsa` (which is equiv to `ssh-keygen -t rsa -b 2048`)